pycrostates/segmentation/segmentation.py

```diff
@@ -173,3 +248,8 @@ def plot(
     def epochs(self) -> BaseEpochs:
         """`~mne.Epochs` instance from which the segmentation was computed."""
         return self._inst.copy()
+
+    @property
+    def metadata(self) -> Optional[DataFrame]:
+        """Epochs metadata."""
+        return self._inst.metadata
```

Comment:
TODO: remove and edit docstring above, we should probably not start exposing all methods from an epoch object here.